12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,4 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
+#!/usr/bin/python
 
+a_to_z = [str(chr(x)) for x in range(97,123)]
+print a_to_z
```

Comment:
Or you can use python all the way and let it figure out 'a' and 'z' ascii codes too:

```
a_to_z = [str(chr(x)) for x in range(ord('a'),ord('z')+1)]
```